### DISCUSSION TITLE
Add documentation regarding Cloudfront

### DISCUSSION THREAD

**Discussion by peimansh** [craftcms/aws-s3#127] [Ideas]
### Description

Would be great to explain more about Cloudfront configurations and permissions that are needed in order to work properly.
In the README, only permissions for S3 are explained.
I didn't know about the required permissions and invalidations, etc.. were not working as expected.
I think these are the least permissions required:

cloudfront:GetDistribution
cloudfront:GetDistributionConfig
cloudfront:CreateInvalidation
cloudfront:GetInvalidation
cloudfront:ListInvalidations

**Comment by karensg** [comment]
Thanks for posting it here, would be great to add indeed.
